packages/template-tachyons/src/Css.ts

```diff
@@ -1688,6 +1688,31 @@ class CssBuilder<T extends Properties1> {
     }
   }
 
+  get ifPrint() {
+    return this.newCss({ selector: "@media print" });
+  }
+  get ifSm() {
+    return this.newCss({ selector: "@media screen and (max-width:599px)" });
+  }
+  get ifMd() {
```

Comment:
Question - do these breakpoints need to stay in lockstep with these in beam?
https://github.com/homebound-team/beam/blob/main/src/Css.ts#L1122

For example, medium is 600px to 1024px

Reply:
Ah yeah, good question @blambillotte ; Beam has it's own config:

https://github.com/homebound-team/beam/blob/main/truss-config.ts#L124

So it's `Css.ts` will have `if...` methods generated based on that.

This `Css.ts` file is just an example/template `Css.ts` that I check in as a mini-regression test for "did this PR break any codegen things?".